projects/vassily-kandinsky/index.md

```diff
@@ -0,0 +1,156 @@
+--- 
+authors: 
+  - "Paulina Mrksic"
+authors-url: 
+  - "https://github.com/paulinalinaa"
+layout: project
+title: "Vassily Kandinsky"
+type: project
+---
+
+
+
+# Vassily Kandinsky 
```

Comment:
the title of the project will be added by the system.  See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax